zerver/lib/export.py

```diff
@@ -1632,7 +1636,11 @@ def _get_exported_s3_record(
     record.update(key.metadata)
 
     if processing_emoji:
-        record["file_name"] = os.path.basename(key.key)
+        file_name = os.path.basename(key.key)
+        # Both the main emoji file and the .original version should have the same
+        # file_name value in the record, as they reference the same emoji.
+        file_name = file_name.removesuffix(".original")
```

Comment:
This seems a bit fragile if a third-party file happens to have had the `.original` suffix?